components/_common_/Form/FormInput/__tests__/FormInput.test.js

```diff
@@ -1,11 +1,88 @@
 /* eslint-env jest */
 import React from 'react';
 import createSnapshotTest from 'test-utils/createSnapshotTest';
+import { mount } from 'enzyme';
 
 import FormInput from '../FormInput';
 
 describe('FormInput', () => {
   test('should render with just required props passed', () => {
     createSnapshotTest(<FormInput id="test" />);
   });
+
+  test('should render with label', () => {
```

Comment:
Awesome tests here on all our validations! This will help when we refactor all of our forms components.